### PR TITLE
264 fix uncaught exception

### PR DESCRIPTION
**Pull request by mastrogiovanni**

### CODE REVIEW

File: api/src/index.ts [hunk 1]
```diff
@@ -46,43 +47,60 @@ async function getRootIdentityId(config: Config): Promise<string> {
 	return null;
 }
 
+process.on("uncaughtException", function(err) {
+    // clean up allocated resources
+    // log necessary error details to log files
+	console.log("ERRORE!!!");
```
Comment (dominic22) [inline api/src/index.ts:53]: I think the logger should be used instead of console.log. in addition there is a typo

File: api/src/index.ts [hunk 2]
```diff
@@ -21,7 +21,8 @@ const argv = yargs
 	.command('server', 'Start the integration service API', {})
 	.command('keygen', 'Generate root identity for integration service API', {})
 	.help()
-	.alias('help', 'h').argv;
+	// .alias('help', 'h')
```
Comment (dominic22) [inline api/src/index.ts:24]: please remove line if obsolete

File: api/src/index.ts (hunk 1 above)
Comment (mastrogiovanni) [reply api/src/index.ts]: done

File: api/src/index.ts (hunk 2 above)
Comment (mastrogiovanni) [reply api/src/index.ts]: done

**Review by dominic22 — APPROVED** [review]

**Pull request merged**